fiatconnect-api.md

```diff
@@ -1796,6 +1796,293 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccount`
+
+`CheckingAccountInr` is a Fiat Account schema that represents accounts for transactions across multiple currencies.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "institutionCode": `string`,
+    "accountNumber": `string`,
+    "fiatAccountType": `FiatAccountTypeEnum.BankAccount`
+}
+```
+
+The `country` field should be a [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. The field is for providers to specify which country this data is meant for.
```

Comment:
I don't see a `country` field in the schema.

Reply:
resolved